packages/test-vectors/src/generators/LEIevidenceDocument.js

```diff
@@ -0,0 +1,93 @@
+const faker = require('faker');
+
+const getLEIevidenceDocument = () => {
+  const example = {
+    '@context': ['https://w3id.org/traceability/v1'],
+    type: 'LEIevidenceDocument',
+    "data": {
+      "type": faker.lorem.word(),
+      "id": faker.random.uuid(),
+      "attributes": {
+        "lei": faker.lorem.word(),
+        "entity": {
+          "legalName":  {
+            "name": faker.company.companyName(),
+            "language": null
+          },
+          "otherNames": [],
+          "transliteratedOtherNames": [],
+          "legalAddress":  {
+            "language":  null,
+            "addressLines": [],
+            "addressNumber": null,
+            "addressNumberWithinBuilding": null,
+            "mailRouting": null,
+            "city": faker.address.city(),
+            "region": faker.address.county(),
+            "country":  faker.address.country(),
+            "postalCode": faker.address.zipCode()
+          },
+          "headquartersAddress": {
+            "language": null,
+            "addressLines": [],
+            "addressNumber": null,
+            "addressNumberWithinBuilding": null,
+            "mailRouting": null,
+            "city": faker.address.city(),
+            "region": faker.address.county(),
+            "country":  faker.address.country(),
+            "postalCode": faker.address.zipCode()
+          },
+          "registeredAt": {
+            "id": faker.random.uuid(),
+            "other": null
+          },
+          "registeredAs": faker.address.city(),
+          "jurisdiction":  faker.lorem.word(),
+          "category": null,
```

Comment:
why include null?

Reply:
all nullable fields were removed from the "require" property and defined as strings. Category randomly selected from predefined category's array